love_letter_sim2.py

```diff
@@ -80,27 +79,29 @@ def check_hand(self):
                         # JHA - we have to do something different here.
                         self.hand.remove(card)
 
-    def play_card(self,other_player = None):
+    def play_card(self,played_card = None,other_player = None):
```

Comment:
I'm missing what played_card is for?  I'll point you to my repo where I've been working on this a bit....